Templates/QueueTrigger/index.js

```diff
@@ -1,5 +1,4 @@
-﻿module.exports = function (workItem, context) {
-    context.log('Node.js queue trigger function processed work item ' + workItem.id);
-
-    context.done(null, workItem);
+﻿module.exports = function (myQueueItem, context) {
+    context.log('Node.js queue trigger function processed work item ' + myQueueItem.id);
```

Comment:
nit: the "work item" nomenclature is a carry over from my earliest samples. Looks like you've generalized the samples, so perhaps just "processed item" or "processed message" makes more sense?